roles/load_aware_deploy_trimaran/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+- name: Configure user application monitoring
+  command:
+    oc apply -f {{ user_applications_monitor_config }}
+
+- name: Ensure user monitoring enabled
+  shell:
+    set -o pipefail;
+
+    oc -n openshift-user-workload-monitoring get pod
+      | awk 'NR > 1 { print $3}'
+  register: monitoring_enabled
+  delay: 3
+  retries: 20
+  # until all of the pods are in the running state
+  until: "'Completed' not in monitoring_enabled.stdout and 'Failed' not in monitoring_enabled.stdout and 'Pending' not in monitoring_enabled.stdout" 
+
+- name: Deploy Trimaran scheduler
+  shell:
+    set -o errexit;
+    set -o pipefail;
+    set -o nounset;
+    set -o errtrace;
+
+    export MONITORING_SECRET=$(oc get secret -n openshift-user-workload-monitoring
+      | grep  prometheus-user-workload-token
+      | head -n 1
+      | awk '{print $1 }'
+    )
+
+    export MONITORING_TOKEN=$(echo $(oc get secret $MONITORING_SECRET -n openshift-user-workload-monitoring -o json | jq -r '.data.token')
```

Comment:
```
export MONITORING_TOKEN=$(oc get secret $MONITORING_SECRET_NAME -n openshift-user-workload-monitoring -o json | jq -r .data.token | base64 -d)
```
(no need for the nest `$()`)

Reply:
I'll also include this in a PR with general fixes/improvements.

I also found this a bit odd, it actually came from a portion of the OpenShift docs:

> Procedure
> Extract a token to connect to Prometheus:
> ```
> $ TOKEN=`echo $(oc get secret $SECRET -n openshift-user-workload-monitoring -o json | jq -r '.data.token') | base64 -d`
> ```

https://docs.openshift.com/container-platform/4.10/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects